dockerfiles/idp_prod.Dockerfile

```diff
@@ -84,17 +85,18 @@ WORKDIR $RAILS_ROOT
 COPY .ruby-version $RAILS_ROOT/.ruby-version
 COPY Gemfile $RAILS_ROOT/Gemfile
 COPY Gemfile.lock $RAILS_ROOT/Gemfile.lock
+COPY package.json $RAILS_ROOT/package.json
```

Comment:
We probably want to keep the ordering here for the same reasons discussed in https://gitlab.login.gov/lg/identity-dashboard/-/merge_requests/42

Reply:
On this one, I'm going to stand firm, because this is not inline with development, but is aimed at production, and thus the image size reduction is important.

Reply:
Yeah, I wholeheartedly agree that the slimmer image is worthwhile, but we should probably do a multi-stage build since it allows us to maximize the benefits of both caching and smaller builds. The size difference doesn't seem that significant at this point (~5-6%) for a potentially significant increase in build times.

Is it possible to try a multi-stage build?

Reply:
I don't believe the build time is relevant here, though.  It takes 8 minutes to build the prod image.  6 minutes to build the regular image.  The overall pipeline with tests takes 16 minutes because of tests.  The image builds are started at the start of the pipeline, and nothing depends on the prod image, so nothing has to wait for it to complete before it can run, and they run in parallel in a different runner pool, so don't even take up any test slots.

I will dig into the multi-stage builds to see if there's a better way to do it than when I last looked at this, but again, I think that this is not really needed.

Reply:
How's that?  I added a multi-stage build for you!  :-)